etl/controller/pulsemaintainer.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package controller
+
+import (
+	"context"
+	"time"
+
+	"github.com/insolar/insolar/log"
+
+	"github.com/insolar/block-explorer/etl/types"
+)
+
+func (c *Controller) pulseMaintainer(ctx context.Context) {
```

Comment:
Please add some tests for this new functionality

Reply:
ok